<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFormateur</class>
 <widget class="QMainWindow" name="MainFormateur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface Responsable</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/image/responsable.png</normaloff>:/image/responsable.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
 border-image:url(:/image/bg.jpg);

}

#widget{
 border: 1px solid;
background: rgba(0, 0, 0, 80);

}

#label,#label_2,#currentDateTime,#logo,#label_3,#label_5{ color: white; }
QLineEdit,QDateEdit ,#comboBoxRole{ border-radius: 3px; }
#label_13,#label_12,#label_6,#label_8,#label_7,#label_9,#label_10,#label_14,#label_11,#label_4,#labelRole,#labelPassword,#labelConfirmPassword{

color: white;
 
  font-family: Arial;

}



QPushButton
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}

QPushButton:hover { background-color: #66c011; }

#topPanel { background-color: qlineargradient(spread:reflect, x1:0.5, y1:0, x2:0, y2:0, stop:0 rgba(46, 204, 233, 100), stop:1 rgba(32, 80, 96, 5)); 


}


#restartButton{
 border-radius: 15px;

}
#shutdownButton{
 border-radius: 15px;

}

QGroupBox{

color: white;

}



#radioButtonCreer,#radioButtonModifier{

 color: white;

}

#labelUtilisateurConnecteTitre{

color: white;
 
  font-family: Arial;

}

#labelUtilisateurConnecte{

color: white;
 
  font-family: Arial;
}










</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>  Formulaire  d'enregistrement des seances de cours</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>611</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>271</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="comboBox_classe">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#comboBox_classe
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}

#comboBox_classe:hover { background-color: #66c011; }</string>
     </property>
     <item>
      <property name="text">
       <string>Liste Classe</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>58</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string> Matiere : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Classe :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Jour :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Heure :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_jour">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#comboBox_jour
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}

#comboBox_jour:hover { background-color: #66c011; }</string>
     </property>
     <item>
      <property name="text">
       <string>Lundi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mardi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mercredi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jeudi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vendredi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Samedi</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_heure">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#comboBox_heure
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}

#comboBox_heure:hover { background-color: #66c011; }</string>
     </property>
     <item>
      <property name="text">
       <string>1H30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2H</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3H</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4H</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Matiere">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Matricule de Formatuer :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_matr">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>270</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Id :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Id">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>-2</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioButtonCreer">
         <property name="text">
          <string>Creer</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonModifier">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>891</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonValider">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLister">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Lister</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSupprimer">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExit">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonRetour">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>Retour</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
